libs/filamat/src/shaders/ShaderGenerator.cpp

```diff
@@ -254,16 +254,22 @@ const std::string ShaderGenerator::createFragmentProgram(filament::backend::Shad
         case BlendingMode::TRANSPARENT:
             cg.generateDefine(fs, "BLEND_MODE_TRANSPARENT", true);
             break;
+        case BlendingMode::ADD:
+            cg.generateDefine(fs, "BLEND_MODE_ADD", true);
+            break;
+        case BlendingMode::MASKED:
+            cg.generateDefine(fs, "BLEND_MODE_MASKED", true);
+            break;
         case BlendingMode::FADE:
             // Fade is a special case of transparent
             cg.generateDefine(fs, "BLEND_MODE_TRANSPARENT", true);
             cg.generateDefine(fs, "BLEND_MODE_FADE", true);
             break;
-        case BlendingMode::ADD:
-            cg.generateDefine(fs, "BLEND_MODE_ADD", true);
+        case BlendingMode::MULTIPLY:
```

Comment:
There are no equivalent of these constants on the java side?

Reply:
Yeah I realized last night I forgot :)